SUPPORT.md

```diff
@@ -0,0 +1,5 @@
+# Ark Support
```

Comment:
Support.md is a relatively new feature by github: https://help.github.com/articles/adding-support-resources-to-your-project/

It's better than just adding it to the readme because it autogenerates a little popup any time someone goes to file a new issue.